src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        //Test Plane

```

Comment:
please remove commented code before pushing

the purpose of your code should be clear without additional explanation

Reply:
Fixed.